Services/Masterdata/Services.Masterdata.EF/Services.Masterdata.EF.csproj

```diff
@@ -1,8 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <ItemGroup>
+    <None Remove="RawSql\StoredProcedures\sp_GetMasterdataRecursiveTree.sql" />
```

Comment:
add here also the new stored procedure: sp_GetMasterdataFilterByRelatedMasterdata.sql